instruqt-tracks/terraform-intro-gcp/track_scripts/setup-workstation

```diff
@@ -43,9 +43,11 @@ cp ${GITDIR}/exercises/outputs.tf.start ${GITDIR}/outputs.tf
 mv ${GITDIR}/terraform.tfvars.example ${GITDIR}/terraform.tfvars
 
 # Store our project ID as a Terraform env var
-export TF_VAR_project=$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID
```

Comment:
There's no need to export or set this, since we never use it again for the rest of the script.